src/vs/editor/browser/controller/mouseHandler.ts

```diff
@@ -384,6 +386,12 @@ class MouseDownOperation extends Disposable {
 				() => !this._viewHelper.isDirty()
 			)
 		);
+		this._mouseDragThenMoveEventHandler = this._register(
```

Comment:
No need for a separate handler, I suggest to code the dnd case inside `onMouseDownThenMove`